llvm/lib/MC/MCContext.cpp

```diff
@@ -718,10 +719,17 @@ MCSectionCOFF *MCContext::getCOFFSection(StringRef Section,
   StringRef CachedName = Iter->first.SectionName;
   MCSymbol *Begin = getOrCreateSectionSymbol<MCSymbolCOFF>(Section);
   MCSectionCOFF *Result = new (COFFAllocator.Allocate()) MCSectionCOFF(
-      CachedName, Characteristics, COMDATSymbol, Selection, Begin);
+      CachedName, Characteristics, COMDATSymbol, Selection, UniqueID, Begin);
   Iter->second = Result;
   auto *F = allocInitialFragment(*Result);
   Begin->setFragment(F);
+  // Normally the comdat symbol is function begin label and will be set a
```

Comment:
Looks little bit ugly here. Any better solution?
We must support create pseudo_probe_desc section from asm so it can't be simply implemented in getPseudoProbeDescSection